circle.yml

```diff
@@ -0,0 +1,99 @@
+    BUCKET_NAME_CPMC_UC: "circle-dummy-test-cpmc_uc"
+    BUCKET_NAME_CPMC_USC: "circle-dummy-test-cpmc_usc"
+    BUCKET_NAME_NYC_CU: "circle-dummy-test-nyc_cu"
+    BUCKET_NAME_NYC_CORNELL: "circle-dummy-test-nyc_cornell"
+    BUCKET_NAME_NYC_HH: "circle-dummy-test-nyc_hh"
+    BUCKET_NAME_ECCHC: "circle-dummy-test-ecchc"
+    BUCKET_NAME_GHS: "circle-dummy-test-ghs"
+    BUCKET_NAME_HRHC: "circle-dummy-test-hrhc"
+    BUCKET_NAME_IPMC_NU: "circle-dummy-test-ipmc_nu"
+    BUCKET_NAME_IPMC_UCHICAGO: "circle-dummy-test-ipmc_uchicago"
+    BUCKET_NAME_IPMC_NORTHSHORE: "circle-dummy-test-ipmc_northshore"
+    BUCKET_NAME_IPMC_RUSH: "circle-dummy-test-ipmc_rush"
+    BUCKET_NAME_IPMC_UIC: "circle-dummy-test-ipmc_uic"
+    BUCKET_NAME_JHCHC: "circle-dummy-test-jhchc"
+    BUCKET_NAME_NEC_BMC: "circle-dummy-test-nec_bmc"
+    BUCKET_NAME_NEC_PHS: "circle-dummy-test-nec_phs"
+    BUCKET_NAME_PITT: "circle-dummy-test-pitt"
+    BUCKET_NAME_SYHC: "circle-dummy-test-syhc"
+    BUCKET_NAME_TACH_HFHS: "circle-dummy-test-tach_hfhs"
+    BUCKET_NAME_UAMC_BANNER: "circle-dummy-test-uamc_banner"
+    BUCKET_NAME_UAMC_UOFA: "circle-dummy-test-uamc_uofa"
+    BUCKET_NAME_VA: "circle-dummy-test-va"
+    BUCKET_NAME_AOUW_MCRI: "circle-dummy-test-aouw_mcri"
+    BUCKET_NAME_AOUW_UWH: "circle-dummy-test-aouw_uwh"
+    BUCKET_NAME_AOUW_MCW: "circle-dummy-test-aouw_mcw"
+    BUCKET_NAME_SEEC_UFL: "circle-dummy-test-seec_ufl"
+    BUCKET_NAME_SAOU_UAB: "circle-dummy-test-saou_uab"
+    BUCKET_NAME_SAOU_UAB_HUNT: "circle-dummy-test-saou_uab_hunt"
+    BUCKET_NAME_SAOU_CGMHS: "circle-dummy-test-saou_cgmhs"
+    BUCKET_NAME_SAOU_USAHS: "circle-dummy-test-saou_usahs"
+    BUCKET_NAME_SAOU_UMC: "circle-dummy-test-saou_umc"
+    BUCKET_NAME_SAOU_TUL: "circle-dummy-test-saou_tul"
+    BUCKET_NAME_SAOU_LSU: "circle-dummy-test-saou_lsu"
+    BUCKET_NAME_SAOU_UMMC: "circle-dummy-test-saou_ummc"
+    PYTHONPATH: "/usr/lib/google-cloud-sdk/platform/google_appengine/:~/curation/data_steward:~/curation/data_steward/lib"
+
+  post:
+    - echo "deb http://packages.cloud.google.com/apt cloud-sdk-$(lsb_release -c -s) main" | sudo tee -a /etc/apt/sources.list.d/google-cloud-sdk.list
+    - curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo apt-key add -
+    - sudo apt-get update && sudo apt-get install google-cloud-sdk
+    - sudo apt-get install google-cloud-sdk-app-engine-python
+    - chmod 700 ./curation/data_steward/ci/activate_creds.sh
+    - ./curation/data_steward/ci/activate_creds.sh ~/gcloud-credentials-key.json
+
+test:
+  override:
+    - cd ~/curation/data_steward && ./test/run_tests.sh -g /usr/lib/google-cloud-sdk/platform/google_appengine/ -r spec_test.py
+    - cd ~/curation/data_steward && ./test/run_tests.sh -g /usr/lib/google-cloud-sdk/platform/google_appengine/ -r bq_utils_test.py
+      # - cd ~/curation/data_steward && ./test/run_tests.sh -g /usr/lib/google-cloud-sdk/platform/google_appengine/ -r validation_test.py:
```

Comment:
Are these two lines intended to be commented out?

Reply:
Yet to be decided. I would like to run the tests separately.